src/theme/SearchBar/index.js

```diff
@@ -32,7 +74,8 @@ const Search = props => {
           // So, we can safely remove it. See https://github.com/facebook/docusaurus/issues/1828 for more details.
 
           history.push(url);
-        }
+        },
+        versionsToSearch: versionToSearch ? [versionToSearch] : null,
```

Comment:
If versionToSearch is null, are searched all the versions?

Reply:
yes, they are